python/pyarrow/src/CMakeLists.txt

```diff
@@ -59,7 +61,6 @@ endif()
 #
 
 find_package(Arrow REQUIRED)
-include(ArrowOptions)
```

Comment:
For my own understanding, the reason this line is no longer needed, is because this will also be included by `find_package(Arrow)` already? (since `ArrowConfig.cmake` has a line `include("${CMAKE_CURRENT_LIST_DIR}/ArrowOptions.cmake")`)

Reply:
Right.